core/src/main/java/org/infinispan/manager/DefaultCacheManager.java

```diff
@@ -821,7 +822,8 @@ private void internalStop() {
          stopCaches();
          globalComponentRegistry.getComponent(CacheManagerJmxRegistration.class).stop();
          globalComponentRegistry.stop();
-         log.debugf("Stopped cache manager %s", configurationManager.getGlobalConfiguration().transport().nodeName());
+         log.debugf("Stopped cache manager %s on %s", globalConfiguration.cacheManagerName(),
+                    globalConfiguration.transport().nodeName());
```

Comment:
why not using the `getAddress()`?

Reply:
I think there was a time when `JGroupsTransport.stop()` set the address to `null`.

A more actual reason is that I made `TestCacheManagerFactory` set the node name even when the test is local, so the node name is set even for local cache managers. Although I realize that tests using local cache managers are very unlikely to use more than one cache manager, so there is no extra information in the node name.

Reply:
you could have stored the `getAddress()` before invoking `stop()` :)